validator/lib/validateB3dm.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var Cesium = require('cesium');
+var DeveloperError = Cesium.DeveloperError;
+module.exports = validateB3dm;
```

Comment:
Add empty lines between these three, this will provide better styling once we start to include more libraries. Look at `readTile` for example.